src/single-spa-angularjs.js

```diff
@@ -26,12 +26,6 @@ export default function singleSpaAngularJS(userOpts) {
     throw new Error(`single-spa-angularjs must be passed opts.angular`);
   }
 
-  if (opts.domElementGetter && typeof opts.domElementGetter !== "function") {
```

Comment:
This type check was [moved down here](https://github.com/single-spa/single-spa-angularjs/pull/50/files#diff-f7bde57554914970afa34440a0daa7b5R138-R143) 👇  in order to capture the `domElementGetter` hooks more efficiently.

Because the React version requires all element getters to be a functions, where the previous version had a mixture of DOM elements and functions as their references.